test/index.test.html

```diff
@@ -0,0 +1,26 @@
+<!--
+Copyright (c) 2017 Google Inc. All rights reserved.
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt
+Code distributed by Google as part of this project is also
+subject to an additional IP rights grant found at
+http://polymer.github.io/PATENTS.txt
+-->
+<!DOCTYPE html>
+<head>
+  <meta charset="utf-8">
+  <title>Arcs CDN Mocha Tests</title>
+  <link href="../node_modules/mocha/mocha.css" rel="stylesheet" />
+</head>
+<div id="mocha"></div>
+<div id="target"></div>
+<script src="../node_modules/chai/chai.js"></script>
+<script src="../node_modules/mocha/mocha.js"></script>
+<script>mocha.setup('bdd')</script>
+
+<script src="../v0.2.1/apps/chrome-extension/data-processing.js"></script>
```

Comment:
Interesting. Somehow I had envisioned we'd only have tests built on `../dev/...` so that we don't, for example, have to block copy and paste editing version every time we freeze a version. And since keeping ToT non-broken is primary risk since post-freeze in theory old versions don't break.

But I know right now extension is maybe only operational in 0.2.1 so this is fine as a first step in any case but wanted to mention my thoughts for further consideration.

Reply:
That's a good point. I expect there's value in testing the last version as well, since we'll be making some changes (although more slowly) there.

I agree the copy/paste a block when cutting a new version would be bad. I was thinking that we'd end up scoping index.test.html to each directory, and changing the runner to look at all of them (or maybe the most recent of each branch). Would that make sense?

Reply:
Yes, sounds reasonable to me.